AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -345,6 +352,16 @@
         ]
       }
     },
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
+      "apiVersion": "2022-04-01",
+      "name": "[parameters('roleNameGuid')]",
+      "scope": "[parameters('azureMonitorWorkspaceResourceId')]",
+      "properties": {
+          "roleDefinitionId": "[concat('/subscriptions/', variables('clusterSubscriptionId'), '/providers/Microsoft.Authorization/roleDefinitions/', 'acdd72a7-3385-48ef-bd42-f606fba81ae7')]",
+          "principalId": "[reference(resourceId('Microsoft.Dashboard/grafana', split(parameters('grafanaResourceId'),'/')[8]), '2022-08-01', 'Full').identity.principalId]"
```

Comment:
Can you also create a PR to update the ARM template documenation to remove the extra role assignment step and mention the permission level required for customers to enable this?

Reply:
Sure! I am doing this in the next commit in the prometheus-collector repo. I will create a separate pr for updating the public docs in azure-pr-docs repo.